scripts/postinst.in

```diff
@@ -68,6 +68,14 @@ if [ $? != 0 ]; then
   valid $? "Error adding pm2 to startup."
 fi
 
+# Create a terrama2 user
+# and default data folder
+(
+  adduser --system terrama2
```

Comment:
This command seems to create user home. Is it necessary? Maybe giving option `--no-create-home`.